plugins/Organic/Organic.cpp

```diff
@@ -159,61 +155,59 @@ OrganicInstrument::~OrganicInstrument()
 
 
 
-void OrganicInstrument::saveSettings( QDomDocument & _doc, QDomElement & _this )
+void OrganicInstrument::saveSettings(QDomDocument& doc, QDomElement& elem)
 {
-	_this.setAttribute( "num_osc", QString::number( m_numOscillators ) );
-	m_fx1Model.saveSettings( _doc, _this, "foldback" );
-	m_volModel.saveSettings( _doc, _this, "vol" );
+	elem.setAttribute("num_osc", QString::number(m_numOscillators));
+	m_fx1Model.saveSettings(doc, elem, "foldback");
+	m_volModel.saveSettings(doc, elem, "vol");
 
-	for( int i = 0; i < m_numOscillators; ++i )
+	for (int i = 0; i < m_numOscillators; ++i)
 	{
-		QString is = QString::number( i );
-		m_osc[i]->m_volModel.saveSettings( _doc, _this, "vol" + is );
-		m_osc[i]->m_panModel.saveSettings( _doc, _this, "pan" + is );
-		m_osc[i]->m_harmModel.saveSettings( _doc, _this, "newharmonic" + is );
-
-		m_osc[i]->m_detuneModel.saveSettings( _doc, _this, "newdetune"
-									+ is );
-		m_osc[i]->m_oscModel.saveSettings( _doc, _this, "wavetype"
-									+ is );
+		QString is = QString::number(i);
```

Comment:
Change to be consistent with code elsewhere.
```suggestion
		const auto is = QString::number(i);
```